src/hooks/useDataSync/useDataSync.tsx

```diff
@@ -85,16 +85,16 @@ export const useDataSync: UseDataSync = () => {
   }
 
   const resetCaches = () => {
+    const now = Date.now()
     setSyncTimestamps({
-      [DataModel.BALANCE_SHEET]: initialTimestamp,
-      [DataModel.CHART_OF_ACCOUNTS]: initialTimestamp,
-      [DataModel.JOURNAL]: initialTimestamp,
-      [DataModel.LEDGER_ACCOUNTS]: initialTimestamp,
-      [DataModel.LINKED_ACCOUNTS]: initialTimestamp,
-      [DataModel.PROFIT_AND_LOSS]: initialTimestamp,
-      [DataModel.STATEMENT_OF_CASH_FLOWS]: initialTimestamp,
+      [DataModel.BALANCE_SHEET]: now,
```

Comment:
Setting these to the `initialTimestamp` makes it an _earlier_ time than now - when what we really want is to think that the data has been synced more _recently_.